internal/ctxval/ctx_getters.go

```diff
@@ -37,3 +38,10 @@ func GetRequestId(ctx context.Context) string {
 func GetIdentity(ctx context.Context) identity.XRHID {
 	return identity.Get(ctx)
 }
+
+func GetAccount(ctx context.Context) *models.Account {
+	if ctx.Value(AccountCtxKey) == nil {
+		return nil
+	}
+	return ctx.Value(AccountCtxKey).(*models.Account)
+}
```

Comment:
In the future, I think we should move also all "setters" here. Not as part of this PR, I actually introduced this and now I feel like it would be cleaner to have it here.

Reply:
Yeah, it deserves some discussion I believe, but yeah, having the setters there and making the keys private is probably a good path forward.